homeassistant/components/enocean/binary_sensor.py

```diff
@@ -82,9 +82,9 @@ def value_changed(self, packet):
         # Energy Bow
         pushed = None
 
-        if packet.data[6] == 0x30:
+        if packet.data[6] == 0x30 or packet.data[6] == 0x31:
```

Comment:
We don't allow changes to this part of the code as it belongs in a 3rd party library. It's protocol details.